lib/ansible/vars/__init__.py

```diff
@@ -150,6 +153,9 @@ def extra_vars(self, value):
     def set_inventory(self, inventory):
         self._inventory = inventory
 
+    def set_check(self, check):
```

Comment:
I can think of two ways to do this and since I'm very new to the Ansible code, what do you think would be better:

1. Create a new property like `extra_vars`, named e.g. `options_magic_vars` and then create a method in `ansible.utils.vars` which will be similar to `load_extra_vars(loader, options)`, e.g. `load_options_magic_vars(options)` (I guess I wouldn't need a loader here) which will return a dictionary of options for which we want to create magic vars. So in future we would only need to change that one method to add a new magic variable from options?
2. Create a `set_options(self, options)` which will pass all the options to VairableManager in which we can define which options need to become magic variables, so again we will define it in only one place?

Which one do you think fits better, or do you have a better suggestion?